analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DisablingRequestValidation.vb

```diff
@@ -0,0 +1,109 @@
+﻿Imports System
+Imports System.Web.Mvc
+
+Namespace Tests.Diagnostics
+    <ValidateInput(False)> ' Noncompliant {{Make sure disabling ASP.NET Request Validation feature is safe here.}}
+    Public Class NonCompliantClass
+    End Class
+
+    Public Class NonCompliantMethods ' we don't care if it derives from controller
+
+        <ValidateInput(False)> Public Function Foo(ByVal input As String) As ActionResult ' Noncompliant {{Make sure disabling ASP.NET Request Validation feature is safe here.}}
+'        ^^^^^^^^^^^^^^^^^^^^
+            Return Nothing
+        End Function
+
+        <CLSCompliant(False)>
+        <ValidateInput(False)> ' Noncompliant
+        Public Function WithTwoFalse(ByVal input As String) As ActionResult
+            Return Foo(input)
```

Comment:
_Just a note (no action required):_ Function in VB doesn't have to return a value. So the test cases could be just
```
<ValidateInput(False)>
Public Function Something(....
End Function
```